specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2022-11-01-preview/agfood.json

```diff
@@ -12840,6 +12895,126 @@
         }
       }
     },
+    "/scenes/collections/{collectionId}/features/{featureId}": {
+      "get": {
+        "tags": [
+          "Scenes"
+        ],
+        "description": "Get a feature(STAC Item) for given collection and feature id.",
+        "operationId": "Scenes_GetStacFeature",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "collectionId",
+            "description": "Collection Id to be fetched. Available Values are: Sentinel_2_L2A, Sentinel_2_L1C.",
+            "required": true,
+            "type": "string"
```

Comment:
similar comment, prefer extensible enum

Reply:
This is a path parameter. Not sure if we can have enums in path parameters. We use Swashbuckle.AspNetCore Cli Tool and its not generating the swagger properly if we have enums in path parameters, but works fine if its in POST/GET body.

Reply:
I think we can.

The problem might be Swashbuckle. Generally it is not recommended. If it is used, there likely to be some manual modification on the output (like this case).